_posts/2012-02-26-anthem-of-a-new-blog.markdown

```diff
@@ -38,7 +38,7 @@ Next step - a MacBook Air.
 
 **Blog  Content**
 
-I wanna spend few words on what I'll cover here; the blog will be focused on Apple and technology in general, but I've decided to not about rumors or daily news, there'll be articles about news of course, but that's not the main purpose of the blog.
+I wanna spend few words on what I'll cover here; the blog will be focused on Apple and technology in general, but I've decided to not about rumors or daily news, there will be articles about news of course, but that's not the main purpose of the blog.
 
 I want to focus on a little more than that: I'll write commentaries and short 'essays' - I'm not so self-absorbed to presume that I'm able to write a full essay, but just a sort of.
```

Comment:
What ability to write longer form has to being self-absorbed?

Reply:
No idea, this post is from 2012 :)

Reply:
Marked as resolve even though I didn't change a single word here, let me know if I should have xD